internal/publish/manifest.go

```diff
@@ -20,14 +20,14 @@ func (p *defaultPublisher) createManifest() (*bundles.Manifest, error) {
 
 		// Prefer a configured package file when present; otherwise
 		// use the interpreter's discovery (which falls back to renv.lock).
-		if p.Config.R.PackageFile != "" {
-			lockExists, err := p.Dir.Join(p.Config.R.PackageFile).Exists()
-			scanDependencies = (err != nil) || !lockExists
-			if err != nil {
-				p.log.Debug("Error checking existence of configured R lockfile", "lockfile", p.Config.R.PackageFile, "error", err)
-			}
-		} else {
-			scanDependencies = true
+		lockExists, err := p.Dir.Join(p.Config.R.GetPackageFile()).Exists()
```

Comment:
Should this care if the package file is selected as included in the configuration?

If I have a local `renv.lock`, but I don't have selected I think I would expect to see behavior described below:

> Deploying a project with NO existing renv.lock file. An automatically generated renv.lock should exist under .posit/deployments/..., manifest should use R package descriptions from it.